{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -1,28 +1,35 @@
-FROM ubuntu:bionic
+FROM python:3.8-slim-buster as base
 
-ENV LANG C.UTF-8
-ARG DEBIAN_FRONTEND=noninteractive
-# Allow SECRET_KEY to be passed via arg so collectstatic can run during build time
-ARG SECRET_KEY
-
-# libpq-dev and python3-dev help with psycopg2
 RUN apt-get update \
-  && apt-get install -y python3.7-dev python3-pip libpq-dev curl \
-  && apt-get clean all \
+  && apt-get install -y --no-install-recommends python3.7-dev python3-pip libpq-dev gcc \
+  && apt-get purge -y --auto-remove -o APT::AutoRemove::RecommendsImportant=false \
   && rm -rf /var/lib/apt/lists/*
   # You can add additional steps to the build by appending commands down here using the
   # format `&& <command>`. Remember to add a `\` at the end of LOC 12.
   # WARNING: Changes to this file may cause unexpected behaviors when building the app.
   # Change it at your own risk.
 
 WORKDIR /opt/webapp
+
+COPY Pipfil* ./
+RUN pip3 install --no-cache-dir cookiecutter -q 'pipenv==2018.11.26' \
+    && pipenv install --deploy --system
+
+FROM base as release
+
+WORKDIR /opt/webapp
+# Run the web server on port $PORT
+# (Done early to avoid running it on every code change)
+CMD waitress-serve --port=$PORT {{cookiecutter.project_slug}}.wsgi:application
+
 COPY . .
-RUN pip3 install --no-cache-dir -q 'pipenv==2018.11.26' && pipenv install --deploy --system
+COPY --from=base /usr/local/lib/python3.8/site-packages/ /usr/local/lib/python3.8/site-packages/
+COPY --from=base /usr/local/bin/ /usr/local/bin/
+
+
+ARG SECRET_KEY
 RUN python3 manage.py collectstatic --no-input
 
 # Run the image as a non-root user
 RUN adduser --disabled-password --gecos "" django
 USER django
```

Comment:
This should be done before `waitress-serve` so it's served using the `django` user

Reply:
fixed